src/cluster_legacy.c

```diff
@@ -1708,6 +1708,17 @@ void setClusterNodeToInboundClusterLink(clusterNode *node, clusterLink *link) {
     serverAssert(!node->inbound_link);
     node->inbound_link = link;
     link->node = node;
+    if (server.verbosity <= LL_VERBOSE) {
+        char ip[NET_IP_STR_LEN];
+        int port;
+        if (connAddrPeerName(link->conn, ip, sizeof(ip), &port) != -1) {
+            serverLog(LL_VERBOSE, "Bound cluster node %.40s (%s) to connection of client %s:%d",
+                      node->name, node->human_nodename, ip, port);
+        } else {
+            serverLog(LL_VERBOSE, "Error resolving the inbound connection address of node %.40s (%s)",
+                      node->name, node->human_nodename);
+        }
+    }
```

Comment:
Keep it at DEBUG like others ?
```suggestion
    if (server.verbosity <= LL_DEBUG) {
        char ip[NET_IP_STR_LEN];
        int port;
        if (connAddrPeerName(link->conn, ip, sizeof(ip), &port) != -1) {
            serverLog(LL_DEBUG, "Bound cluster node %.40s (%s) to connection of client %s:%d",
                      node->name, node->human_nodename, ip, port);
        } else {
            serverLog(LL_DEBUG, "Error resolving the inbound connection address of node %.40s (%s)",
                      node->name, node->human_nodename);
        }
    }
```

Reply:
This is aligned with the logging level in existing function `clusterAcceptHandler`:

```
/* Use non-blocking I/O for cluster messages. */
serverLog(LL_VERBOSE, "Accepting cluster node connection from %s:%d", cip, cport);

/* Accept the connection now.  connAccept() may call our handler directly
 * or schedule it for later depending on connection implementation.
 */
if (connAccept(conn, clusterConnAcceptHandler) == C_ERR) {
    if (connGetState(conn) == CONN_STATE_ERROR)
        serverLog(LL_VERBOSE, "Error accepting cluster node connection: %s", connGetLastError(conn));
    connClose(conn);
    return;
}
```

which shows there's a new inbound connection, and my added logging will tell which cluster node that connection corresponds to.

And I think VERBOSE is indeed the reasonable logging level here because accepting new connection isn't frequent (usually at cluster startup and reconnection due to server down), unlike cluster packet processing (where DEBUG logging level is definitely appropriate).

Reply:
Sure. I don't mind much. I generally flip flop between DEBUG / NOTICE.